tutorials/HDF5/Subset-HDF5-hyperspectral.Rmd

```diff
@@ -0,0 +1,322 @@
+---
+syncID: 884cc5915a3842c1ab6617b060797fab
+title: "Subsetting NEON HDF5 hyperspectral files to make a smaller deliverable"
```

Comment:
Is "deliverable" a common enough term?  What about "to reduce file size"?

Reply:
Good suggestion. I have made this correction.